### PR TITLE
Feature/contributor repository

### LINKED ISSUE #22: Implement "Contributor" screen without design
Blocked by #21 

## Overview (Required)
Since we need to display the information, we need to create classes for ViewModel, Repository, and API for that.
Please refer to the existing implementation and implement it.
This time there is no need for DB cache.
The API documentation can be found below.
## Links
https://droidkaigi.stoplight.io/docs/ssot-api/openapi/reference/external.v1.yaml

### PR DESCRIPTION
**Pull request by RyuNen344**
## Issue
- #22

## Overview (Required)
- add implementation of api and repository
- api response is fake ☞ [url](https://github.com/DroidKaigi/conference-app-2021/issues/22#issuecomment-791908649)

## Links
-

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by takahirom** [comment]
👀

File: data/api/src/commonMain/kotlin/io/github/droidkaigi/feeder/data/KtorContributorApi.kt [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import io.github.droidkaigi.feeder.data.response.ContributorResponse
+import io.github.droidkaigi.feeder.data.response.ContributorsResponse
+import io.ktor.client.HttpClient
+
+open class KtorContributorApi(
+    private val authApi: AuthApi,
+    private val httpClient: HttpClient,
+) : ContributorApi {
+    override suspend fun fetch(): List<Contributor> = authApi.authenticated {
+        // TODO: 2021/03/06 remove comment out when api implemented
```
Comment (takahirom) [inline data/api/src/commonMain/kotlin/io/github/droidkaigi/feeder/data/KtorContributorApi.kt:13]: Thank you for implementing the API even though it's not yet available! 👍 
Can you make the Fake API and change the test to work using the Fake API?
I think FeedViewModelTest will be helpful.
https://github.com/DroidKaigi/conference-app-2021/blob/f6d311de4fb522d412f5786022474305b7048ac0/android/src/test/java/io/github/droidkaigi/feeder/FeedItemViewModelTest.kt#L112-L118

Comment (takahirom) [reply data/api/src/commonMain/kotlin/io/github/droidkaigi/feeder/data/KtorContributorApi.kt:13]: It is okay to implement it with this PR.
https://github.com/DroidKaigi/conference-app-2021/pull/299

Comment (RyuNen344) [reply data/api/src/commonMain/kotlin/io/github/droidkaigi/feeder/data/KtorContributorApi.kt:13]: sorry, I understood it to create fake response.
ok! I can make it!

Comment (takahirom) [reply data/api/src/commonMain/kotlin/io/github/droidkaigi/feeder/data/KtorContributorApi.kt:13]: Sorry 🙇

Comment (RyuNen344) [reply data/api/src/commonMain/kotlin/io/github/droidkaigi/feeder/data/KtorContributorApi.kt:13]: >It is okay to implement it with this PR.

ok, I will make it on #299

**Review by takahirom — APPROVED** [review]
Thank you for proceeding!

**Pull request merged**